vql/darwin/xattr.go

```diff
@@ -79,8 +79,12 @@ func (self *XAttrFunction) getAttributeValues(
 func (self XAttrFunction) Info(
 	scope vfilter.Scope, type_map *vfilter.TypeMap) *vfilter.FunctionInfo {
 	return &vfilter.FunctionInfo{
-		Name:     "xattr",
-		Doc:      "query a file for the specified extended attribute",
+		Name: "xattr",
+		Doc: "Query a file for the specified extended attribute. " +
```

Comment:
This should be only one line.

The rest of the paragraph can go in the vql.yaml file